File: src/main/kotlin/ympa_kotlin_wiremock_server/models/FeedbackListDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class FeedbackListDTO(
    @field:JsonProperty("feedbackList")
    val feedbackList: kotlin.collections.List<FeedbackDTO>,

    @field:JsonProperty("paging")
    val paging: ScrollingPagerDTO? = null,

)

File: src/App/DTO/FeedbackListDTO.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Отзывы пользователей Яндекс Маркета об указанном магазине.
 */
class FeedbackListDTO
{
    /**
     * @DTA\Data(field="paging", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\ScrollingPagerDTO::class})
     * @DTA\Validator(name="TypeCompliant", options={"type":\App\DTO\ScrollingPagerDTO::class})
     * @var \App\DTO\ScrollingPagerDTO|null
     */
    public $paging;

    /**
     * Список отзывов.  Содержит не более 20 отзывов.
     * @DTA\Data(field="feedbackList")
     * @DTA\Strategy(name="Object", options={"type":::class})
     * @DTA\Validator(name="TypeCompliant", options={"type":::class})
     * @var \App\DTO\FeedbackDTO[]|null
     */
    public $feedback_list;

}
